cases/managers.py

```
@@ -103,3 +103,44 @@ def search(
     def submitted(self):
         draft = get_case_status_by_status(CaseStatusEnum.DRAFT)
         return self.get_queryset().exclude(status=draft)
+
+    def get_application(self, case):
+        from applications.models import StandardApplication
+
+        try:
+            return StandardApplication.objects.get(baseapplication_ptr__case_ptr=case)
+        except StandardApplication.DoesNotExist:
+            pass
+
+        from applications.models import OpenApplication
+
+        try:
+            return OpenApplication.objects.get(baseapplication_ptr__case_ptr=case)
+        except OpenApplication.DoesNotExist:
+            pass
```

his method should raise an exception if the case is not found in Standard or Open applications